### DISCUSSION TITLE
Version 1.8.0 and RAG

### DISCUSSION THREAD

**Discussion by koayst-rplesson** [FlowiseAI/Flowise#2494] [General]
Recently, I upgraded to version 1.8.0 and using the same ChatFlow saved from version 1.7.2, the ChatFlow cannot run properly. It seems to me that RAG is not working.  I loaded a FAQ text file containing the FAQ, upset to a in-memory vector store and using a conversation retrieval QA chain.  

I asked question similar to those from the FAQ.txt and the response is "I am not sure".

Previous version: Running in Windows 11
-node-v20.12.2-win-x64
-flowise 1.7.2

<img width="425" alt="Screenshot 2024-05-27 133726" src="https://github.com/FlowiseAI/Flowise/assets/167511739/ff26009d-ac05-4318-8974-ddacdf5108d3">

Attached a screenshot of the flowchat.

What could be the issue?

What change in version 1.8.0 that can cause the chatflow not working properly

**Comment by trunghuynh228** [comment]
I'm having the same issue. Even I use RAG or agent, the chat cannot retrieve information anymore

**Comment by koayst-rplesson** [comment]
I think version 1.8.0 is crippled ... :-(

Anyone from flowise can confirm a new version will be releasing soon ?

**Comment by automaton82** [comment]
https://github.com/FlowiseAI/Flowise/issues/2501 probably related
